src/grb_constrs.jl

```diff
@@ -214,6 +232,75 @@ function add_sos!(model::Model, sostype::Symbol, idx::Vector{Int}, weight::Vecto
         throw(GurobiError(model.env, ret))
     end
 end
+function del_sos!(model::Model, idx::Vector{Cint})
+    # int XPRS_CC XPRSdelsets(XPRSprob prob, int ndelsets, const int mindex[])
+    numdel = length(idx)
+    ret = @grb_ccall(delsos, Cint, (
+                     Ptr{Void},
+                     Cint,
+                     Ptr{Cint}),
+                     model, convert(Cint,numdel), ivec(idx-1) )
+    if ret != 0
+        throw(GurobiError(model.env, ret))
+    end
+end
+
+get_sos_matrix(model::Model) = get_sos(model::Model, 1, num_sos(model))
+function get_sos(model::Model, start::Integer, len::Integer)
+    # int GRBgetsos ( GRBmodel *model,int	*nummembersP,int *sostype,int *beg,int	*ind,doule *weight,int start,int len )
```

Comment:
Also seems out of place, since we don't have comments like these in other wrappers